airflow/providers/google/cloud/utils/credentials_provider.py

```diff
@@ -121,7 +96,7 @@ def provide_gcp_credentials(
             conf_file.flush()
             key_file_path = conf_file.name
         if key_file_path:
-            with temporary_environment_variable(CREDENTIALS, key_file_path):
```

Comment:
I don't understand why it worked before. CREDENTIALS is a variable, and here it is passed as the name of the argument.

Reply:
CREDENTIALS is environment variable and was passed here as `variable_name`